### ISSUE TITLE
Can you provide the configuration for train flower-102

### ISSUE THREAD

**Issue by Johnson-yue** [lucidrains/stylegan2-pytorch#63]
I want to get your flower result as ReadMe ， so I want to train flower102 dataset ， but I do not your experience configuration

**Comment by lucidrains** [comment]
you can download it here https://www.robots.ox.ac.uk/~vgg/data/flowers/

**Comment by Johnson-yue** [comment]
@lucidrains Thank you for giving me the link of dataset, but I don't want to it.
I want to know your train-parameter , like :  network_capacity =?, cl_reg = ?....
and so on

**Comment by lucidrains** [comment]
`--image-size 256 --batch-size 2 --gradient-accumulate-every 8` was enough for me. you can try `--cl-reg`, which is an experimental feature
